tests/conftest.py

```diff
@@ -74,11 +77,21 @@ def spawn_backend(request):
     return request.config.option.spawn_backend
 
 
+class CIEnvoirment(enum.Enum):
+    Travis = 'TRAVIS'
+    Github = 'GITHUB'
+
+
 @pytest.fixture(scope='session')
-def travis():
-    """Bool determining whether running inside TravisCI.
+def ci_env() -> Optional[CIEnvoirment]:
+    """Detect CI envoirment.
     """
-    return os.environ.get('TRAVIS', False)
+    if os.environ.get('TRAVIS'):
+        return CIEnvoirment.Travis
```

Comment:
Hmm, so looking at the [logic below](https://github.com/goodboy/tractor/pull/152/files#diff-d3eea2f9e263e7cd8e79cc7c81011ceaR208) it seems you really are just setting a `bool` yah?

Do we really need an `Enum` type for this?

Reply:
Did you mean "environment" instead?

Reply:
@pquentin this code got replaced I'm pretty sure ;)